src/main/java/Duke.java

```diff
@@ -5,11 +5,17 @@
     private static Task[] tasks;
     private static int taskCount;
 
-    public static void printOutputFormat(String output) {
```

Comment:
Hey I like how you created a method for the output format!

Reply:
Same!